<?xml version="1.0"?>
<runHistory script="keyDown">
 <run>
  <RunDate>2014-12-22 17:03:46.206 +0100</RunDate>
  <Status>Running</Status>
  <Duration>nan</Duration>
  <Errors></Errors>
  <Warnings></Warnings>
  <Exceptions></Exceptions>
  <Successes></Successes>
  <LogFile>20141222_170346.206/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2014-12-22 17:08:50.717 +0100</RunDate>
  <Status>Running</Status>
  <Duration>nan</Duration>
  <Errors></Errors>
  <Warnings></Warnings>
  <Exceptions></Exceptions>
  <Successes></Successes>
  <LogFile>20141222_170850.717/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2014-12-22 17:09:38.732 +0100</RunDate>
  <Status>Failure</Status>
  <Duration>0.354000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20141222_170938.732/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>STMaxCallDepthExceeded -- Handlers were called to a depth exceeding the maximum limit (40). HINT: One common cause for this error is a script that has been given the same name as a built-in command or function unintentionally calling itself.</ErrorMessage>
 </run>
< <run>
  <RunDate>2014-12-22 17:20:50.781 +0100</RunDate>
  <Status>Failure</Status>
  <Duration>0.341000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20141222_172050.781/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>STMaxCallDepthExceeded -- Handlers were called to a depth exceeding the maximum limit (40). HINT: One common cause for this error is a script that has been given the same name as a built-in command or function unintentionally calling itself.</ErrorMessage>
 </run>
</runHistory>
